doc/source/reference/arrays.dtypes.rst

```diff
@@ -298,7 +307,6 @@ Type strings
     .. admonition:: Example
 
        >>> dt = np.dtype((void, 10))  # 10-byte wide data block
-       >>> dt = np.dtype((str, 35))   # 35-character string
```

Comment:
the bytes dtype is kind of useless in python3, I don't think it is worth mentioning.